rabbitmq/config/erlang.cookie

```diff
@@ -0,0 +1 @@
+{{cfg.erlang.cookie ~}}
```

Comment:
I understand the reason for not wanting the dot in the filename but it breaks convention for erlang cookies not to have it as a hidden file. http://erlang.org/doc/reference_manual/distributed.html

Reply:
The file gets copied in the run hook from the config directory to the target location. Since we're doing this `cp`, it could be named anything at all, it won't have a bearing on the Erlang VM.

Reply:
That's right. I just meant it is more conventional to have the cookie file be `.erlang.cookie`